common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayWork.scala

```diff
@@ -81,6 +83,10 @@ case class DisplayWork(
     `type` = "List[uk.ac.wellcome.Display.models.DisplayItem]",
     description = "List of items related to this work."
   ) items: Option[List[DisplayItem]] = None,
+  @Schema(
+    description = "Whether the work contains an item with a digital location",
```

Comment:
Minor: isn't the point of this field to hide this detail from the user? Would a better description be "Whether the work contains an item which can be viewed online"?